src/scripts/components/pages/authors.js

```diff
@@ -1,34 +1,40 @@
 import clearDom from '../../helpers/clearDom';
 import renderToDOM from '../../helpers/renderToDom';
 
+const emptyAuthors = () => {
```

Comment:
```suggestion
const emptyAuthors = () => {
  const btnString = '<button class="btn btn-success btn-lg mb-4" id="add-author-btn">Add An Author</button>';

  renderToDOM('#add-button', btnString);
```

We need to make sure the button is added, otherwise, if there are no authors, the user can't add any.

Reply:
Ignore this comment, I see how this works below.